### DISCUSSION TITLE
mcbbs上下载的地图启动错误

### DISCUSSION THREAD

**Discussion by TXSQ2009** [Meloong-Git/PCL#1594] [提问] [answered]
[错误报告-2023-04-07_下午_9.42.17.zip](https://github.com/Hex-Dragon/PCL2/files/11179308/-2023-04-07_._9.42.17.zip)

**Comment by Chiloven945** [comment]
没看出来是什么问题，请问你是直接下载的版本还是导入的地图？

**Reply by TXSQ2009** [reply]
已解决

**Reply by Chiloven945** [reply]
> 已解决

回复一个answer就可以了，不需要每一个都reply的（

**Comment by MoYuan-CN** [comment]
好像是 `Incompatible magic value 1667592308 in class file sun/reflect/BootstrapConstructorAccessorImpl` 的问题？
@alazeprt 请你过来帮忙看一下

**Reply by TXSQ2009** [reply]
已解决

**Comment by WForst-Breeze** [comment]
看起来您在尝试启动一个 1.12.2 的 Minecraft Forge 整合包（1.12.2, Forge 14.23.5.2855, OptiFine G5），您可以上传整合包文件（如果可以的话）以方便社区更快地为您解决问题~

**Reply by TXSQ2009** [reply, marked as answer]
已解决
